packages/flutter/test/material/navigation_bar_test.dart

```diff
@@ -2,6 +2,11 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// This file is run as part of a reduced test set in CI on Mac and Windows
+// machines.
+@Tags(<String>['reduced-test-set'])
+library;
```

Comment:
Regarding the error: You probably have to delete this line. The empty library statement is a new Dart feature that is likely not available with the dart version used by the stable branch.

```suggestion
```